decentralised-identity.md

```diff
@@ -0,0 +1,106 @@
+| Author | TallAmericano |
+| :---- | :---- |
+| Title | Decentralised Identity and the Potential of Self-Sovereign Identities |
+| Headline  | Looking at Decentralised Identities and how individuals own, control, and share their identity information without the need for a central authority. |
+| Cypherpunktimes | link |
+| Publish Date | xx Mar 2024 |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | twitter link |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement - This article looks at decentralised identities, examining how removing central authorities enables users to have greater confidence in the security and integrity of their identity information..
+
```

Comment:
Current Structure:

* Decentralised Identity and the Potential of Self-Sovereign Identities
* Self-Sovereign Identity (SSI)
* Blockchain Technology
* Privacy Preservation: 
* Interoperability
* Eliminating Single Points of Failure
* User-Centric Approach
* Conclusion

A structural idea that is more critical:

* Decentralised Identity - the good the bad the ugly
        * The Potential of Self-Sovereign Identities
        * Various solutions and what they aim to achieve
        * Benefits and drawbacks
* Can blockchain technology be used for digital identities?
        * What are the benefits of using Blockchain
        * What are the potential drawbacks of using Blockchain
* The trade-off - convenience, privacy, security
        * Understanding the single points of failure
        * Can privacy be achieved?
        * Would security be an issue?
        * convenience, the double edge sword
* Conclusion

Reply:
I have added these areas and adjusted the structure. I agree this outline is better.